### PR TITLE
Spec yml i18n example, readme

### PR DESCRIPTION
**Pull request by shuofan**
## Description

Spec yml i18n example, readme

## Checklist

 - [x] I made sure to check the compatibility of the erda version statemented in the action's spec.yml.
 - [x] My change is adequately tested.

### CODE REVIEW

File: actions/api-test/2.0/spec.yml [hunk 1]
```diff
@@ -121,7 +114,28 @@ accessibleAPIs:
...
     key: params
+locale:
+  zh-CN:
+    desc: 执行单个接口测试。上层可以通过 pipeline.yml 编排一组接口测试的执行顺序。
+    displayName: 接口测试
+    formProps.params.label: API 配置
+    params.asserts.desc: 断言。若接口返回成功，但断言失败，最终结果仍为失败
+    params.asserts.struct.arg.desc: 出参名
+    params.asserts.struct.operator.desc: 比较操作。支持的值有：=, !=, >=, <=, >, <, contains,
+      not_contains, belong, not_belong, empty, not_empty, exist, not_exist
```
Comment (Effet) [inline actions/api-test/2.0/spec.yml:129]: broken line

File: actions/api-test/2.0/spec.yml [hunk 2]
```diff
@@ -121,7 +114,28 @@ accessibleAPIs:
...
+    params.asserts.struct.value.desc: 值
+    params.body.desc: 请求体
+    params.body.struct.type.desc: Content-Type。可选值：none, application/json, application/x-www-form-urlencoded
+      等
```
Comment (Effet) [inline actions/api-test/2.0/spec.yml:133]: broken line

File: actions/archive-release/1.0/spec.yml [hunk 3]
```diff
@@ -125,3 +109,24 @@ accessibleAPIs:
   - path: /api/releases/<releaseId>/actions/get-dice
     method: GET
     schema: http
+locale:
+  zh-CN:
+    desc: 从源仓库将制品快照和数据迁移脚本归档到目标（version）仓库。源仓库一般就是当前仓库。 当前仓库的 <.dice|.erda>/migrations
+      归档到目标仓库的 <version>/sqls 。 应用打包发布的制品归档到目标仓库的 <version>/releases/<snapName>/dice.yml
+      。
```
Comment (Effet) [inline actions/archive-release/1.0/spec.yml:116]: broken line

**Review by Effet — CHANGES_REQUESTED** [review]

File: actions/api-test/2.0/spec.yml (hunk 1 above)
Comment (shuofan) [reply actions/api-test/2.0/spec.yml]: Solve all problems

File: actions/publish-api-asset/1.0/spec.yml [hunk 4]
```diff
@@ -2,38 +2,30 @@ name: publish-api-asset
...
+    desc: ${{ i18n.params.asset_id.desc }}
     required: true
   - name: version
-    desc: API 资源版本，需要使用 X.Y.Z 的格式，不填时自动生成
+    desc: ${{ i18n.params.version.desc }}
   - name: spec_path
-    desc: API 描述文档的路径，eg:${java}/swagger/swagger.json
-    required: true
+    desc: ${{ i18n.params.spec_path.desc }}
+    required: true
```
Comment (Effet) [inline actions/publish-api-asset/1.0/spec.yml:23]: tailing space

File: actions/publish-api-asset/1.0/spec.yml [hunk 5]
```diff
@@ -2,38 +2,30 @@ name: publish-api-asset
...
   - name: spec_path
-    desc: API 描述文档的路径，eg:${java}/swagger/swagger.json
-    required: true
+    desc: ${{ i18n.params.spec_path.desc }}
+    required: true    
   - name: runtime_id
-    desc: runtime_id 值，eg:${dice:OUTPUT:runtimeID}
-    required: true
+    desc: ${{ i18n.params.runtime_id.desc }}
+    required: true
```
Comment (Effet) [inline actions/publish-api-asset/1.0/spec.yml:26]: tailing space

File: actions/push-extensions/1.0/spec.yml [hunk 6]
```diff
@@ -12,29 +12,21 @@
...
-    displayName: "extension push tool"
-    desc: "Extension push tool, push Actions and Addons in the extensions repository to the specified Erda platform"
-
 params:
   - name: repos
     type: string_array
-    desc: extensions 仓库列表, 如 ${git-checkout}
-
+    desc: ${{ i18n.params.repos.desc }}
+
```
Comment (Effet) [inline actions/push-extensions/1.0/spec.yml:29]: tailing space

**Review by Effet — CHANGES_REQUESTED** [review]

File: actions/publish-api-asset/1.0/spec.yml (hunk 4 above)
Comment (shuofan) [reply actions/publish-api-asset/1.0/spec.yml]: done

**Review by Effet — APPROVED** [review]

**Pull request merged**